docs/introduction_tutorial/intro_tutorial_part3.ipynb

```diff
@@ -0,0 +1,444 @@
+{
```

Comment:
<p>Charles, will we have a "production" bakery up and running before Friday? What naming conventions should we use for the bakeries?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-forge/pangeo-forge-recipes/pull/307/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>I had not been planning on that, but it is theoretically not a huge lift. Should we prioritize that deployment?</p>

Reply:
If we really want to be doing things right, we will have two versions of everything. "staging" is not the real thing. However, that may not be feasible here.

Reply:
name: "pangeo-ldeo-nsf-earthcube"